search-commons/src/test/java/no/unit/nva/search/models/NewIndexDocumentTest.java

```diff
@@ -0,0 +1,70 @@
+package no.unit.nva.search.models;
+
+import static no.unit.nva.publication.PublicationGenerator.randomString;
+import static no.unit.nva.search.IndexingConfig.objectMapper;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.model.Publication;
+import no.unit.nva.publication.PublicationGenerator;
+import org.junit.jupiter.api.Test;
+
+class NewIndexDocumentTest {
```

Comment:
tests covering the `NewIndexDocument` behavior after the class was moved to the commons module.